daemon/start_windows.go

```diff
@@ -80,7 +88,50 @@ func (daemon *Daemon) getLibcontainerdCreateOptions(container *container.Contain
 		}
 	}
 
-	// Now build the full set of options
+	// Read and add credentials from the security options if a credential spec has been provided.
+	if container.HostConfig.SecurityOpt != nil {
+		for _, sOpt := range container.HostConfig.SecurityOpt {
```

Comment:
sure would be nice if there was a well tested helper function for grabbing k/v pairs out of SecuritryOpt... or rather `[]string`